.github/workflows/tacos_unlock.yml

```
@@ -90,16 +90,20 @@ jobs:
         uses: ./tacos-gha/.github/actions/setup
         with:
           ssh-private-key: ${{ secrets.ssh-private-key }}
-          # Use author's username when unlocking for closed PRs
+          # Use author's username for the following actions
           user: |
             ${{
               true
               && github.event_name == 'pull_request'
-              && github.event.action == 'closed'
+              && (
+                  false
+                  || github.event.action == 'closed'
+                  || github.event.action == 'converted_to_draft'
+              )
 
               && github.event.pull_request.user.login
 
-              || github.actor
+              || github.triggering_actor
             }}
```

Do parens make the implicit ternary any clearer? Not sure.

```suggestion
            ${{
              (
                true
                && github.event_name == 'pull_request'
                && (
                  false
                  || github.event.action == 'closed'
                  || github.event.action == 'converted_to_draft'
                )
                && github.event.pull_request.user.login
              ) || github.triggering_actor
            }}
```

---

Not particularly more than the whitespace, in my eyes.